### ISSUE TITLE
[Accessibility]: `Set Selection Anchor` and `Select from Anchor to Cursor` does not work in a11y terminal buffer

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#19348]
CC -meganrogge

The following does not work in a11y terminal buffer. Would you please support them?

* Set Selection Anchor (Ctrl+K, Ctrl+B)

* Select from Anchor to Cursor (Ctrl+K, Ctrl+K)
